lib/token_scanner.rb

```diff
@@ -85,7 +90,9 @@ def log_expirations
       puts "Context: #{context}"
       results.each do |result|
         expiration = result[2].positive? ? "expires-in: #{result[2]} days" : "expired: #{result[2].abs} days ago"
-        puts "\tconfigmap: #{result[0]}, key: #{result[1]}, #{expiration}."
+        puts "\tconfigmap: #{result[0]}"
+        puts "\t\t key: #{result[1]}, #{expiration}."
+        puts "\t\t audience_id = #{result[3]}, subject_id = #{result[4]}"
```

Comment:
I'd suggest putting these on the same line as above and sticking to `:` instead of `=`, so the output is more easily scannable.

Reply:
Thanks done in https://github.com/artsy/frequency/pull/47/commits/eeb70f3d90f08575477b5cfa170bb37e7b936e11

I realized that I didn't guard against the `aud` or `subject_application` not being set. Is that something I should add? Looking at the history of [ApplicationTrust](https://github.com/artsy/gravity/blob/e45596d06c987f14558d15f00f9f93aff686b50e/app/models/util/application_trust.rb) it seems the structure has changed over time. Wonder if there are JWTs in use that may have differently named fields or values not set.

Reply:
Yes, we added `subject_application` at some point so old tokens are likely still around. However, it looked to me like the code would work with a `nil` value there.